### PR TITLE
Move long tests to full mode only

### PR DESCRIPTION
**Pull request by benjamin-confino**
#build

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/suite/RepeatFaultTolerance.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public static FeatureReplacementAction ft20Features(String server) {
      */
     public static RepeatTests repeatDefault(String server) {
         return RepeatTests.with(mp20Features(server))
-                        .andWith(ft20Features(server));
+                        .andWith(ft20Features(server).fullFATOnly());
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/suite/RepeatFaultTolerance.java]: Can we keep ft20Features in lite and move mp20Features to full instead?

FT 2.0 is the current version, mp20Features includes FT 1.1.

Also, please update the comment to point out that FT 1.1 is only for FULL mode

**Comment by benjamin-confino** [comment]
#build

**Pull request merged**